### DISCUSSION TITLE
Static warm doesnt touch runway routes

### DISCUSSION THREAD

**Discussion by JelleGroenendal** [statamic-rad-pack/runway#603] [Feature Requests]
### Description

When running php please static:warm the runway model urls are not processed

### Steps to reproduce

Make sure you have a model with runway routes enabled
Run php please static:warm

### Environment

Environment
Application Name: Project
Laravel Version: 11.23.5
PHP Version: 8.3.8
Composer Version: 2.7.8
Environment: local
Debug Mode: ENABLED
URL: project.ddev.site
Maintenance Mode: OFF
Timezone: europe/amsterdam
Locale: en

Cache
Config: NOT CACHED
Events: CACHED
Routes: CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: file
Database: mysql
Logs: stackphp
Mail: smtp
Queue: sync
Session: file

Locales
Installed: de, en, nl
LaravelLang\Lang\Plugin: 15.7.4
Locales Version: 2.9.2
Protected: en
Publisher Version: 16.4.0

Livewire
Livewire: v3.5.6

Sentry
Enabled: MISSING DSN
Environment: local
Laravel SDK Version: 4.8.0
PHP SDK Version: 4.9.0
Release: NOT SET
Sample Rate Errors: 100%
Sample Rate Performance Monitoring: NOT SET
Sample Rate Profiling: NOT SET
Send Default PII: DISABLED

Statamic
Addons: 6
Sites: 1
Stache Watcher: Disabled
Static Caching: full
Version: 5.25.0 PRO

Statamic Addons
eminos/statamic-tabs: 1.2.0
jacksleight/statamic-bard-texstyle: 3.3.0
jonassiewertsen/statamic-livewire: 3.8.0
mitydigital/sitemapamic: 3.2.0
statamic-rad-pack/runway: 7.9.3
statamic/eloquent-driver: 4.14.3

Statamic Eloquent Driver
Asset Containers: file
Assets: file
Blueprints: file
Collection Trees: file
Collections: file
Entries: eloquent
Forms: eloquent
Global Sets: eloquent
Global Variables: file
Navigation Trees: file
Navigations: eloquent
Revisions: eloquent
Sites: file
Taxonomies: eloquent
Terms: eloquent
Tokens: file

**Comment by JelleGroenendal** [comment]
Might actually be a feature request it also didnt work in Statamic 4

**Reply by duncanmcclean** [reply]
Yes, it's a feature request. I've converted this into a discussion instead.

**Comment by duncanmcclean** [comment]
This is now possible, as of [Runway v7.11.0](https://github.com/statamic-rad-pack/runway/releases/tag/v7.11.0).
